pkg/app/app.go

```diff
@@ -94,6 +98,16 @@ func DefineStartCommandFlags(kpApp *kingpin.Application, cmd *kingpin.CmdClause)
 		Envar("HELM_IGNORE_RELEASE").
 		StringVar(&HelmIgnoreRelease)
 
+	// Rate limit settings for kube client used by Helm resources monitor.
```

Comment:
Commentary is right, but I can figure out the same from the name of flags or env vars. The `HELM_KUBE_CLIENT_` is a little confusing prefix because it does not affect the helm, actually.

Reply:
HELM_MONITOR_ do the thing.